src/m_Do/m_Do_main.cpp

```diff
@@ -51,7 +51,8 @@ void HeapCheck::CheckHeap1() {
 }
 
 // unused data?
-static u32 lit_2100[] = {0x3F800000, 0x3F800000, 0x3F800000, 0x3F800000, 0x3F800000, 0x3F800000};
+static u32 lit_2100[] = {0x3F800000, 0x3F800000, 0x3F800000};
+static u32 lit_2080[] = {0x3F800000, 0x3F800000, 0x3F800000};
```

Comment:
Unrelated change, this allows `ninja diff` to work again. Before, it was complaining that the symbol sizes were different here and bailing.